### ISSUE TITLE
Great plugin, but...

### ISSUE THREAD

**Issue by lauri865** [pavlikm/pm2-cluster-memory-cache#14]
I just spent 3 hours debugging something breaking in my app that works standalone, but not when run within the app. Completely unrelated error, and no way to trace it. It just said:
`Cannot assign to read-only property 'to'`

Turns out it's the `to-item` polyfill required here:
https://github.com/pavlikm/pm2-cluster-memory-cache/blob/3dde18c3d85173e7fa390387623e0b61f0910a1d/src/repositories/ProcessRepository.js#L2

Worst thing is it doesn't even seem to be used by the lib (don't see any calls to toItem).

Polyfills are EVIL! 😡😡😡

**Comment by pavlikm** [comment]
> I just spent 3 hours debugging something breaking in my app that works standalone, but not when run within the app. Completely unrelated error, and no way to trace it. It just said: `Cannot assign to read-only property 'to'`
> 
> Turns out it's the `to-item` polyfill required here:
> 
> https://github.com/pavlikm/pm2-cluster-memory-cache/blob/3dde18c3d85173e7fa390387623e0b61f0910a1d/src/repositories/ProcessRepository.js#L2
> 
> Worst thing is it doesn't even seem to be used by the lib (don't see any calls to toItem).
> 
> Polyfills are EVIL! 😡😡😡

it is used on line [#76](https://github.com/pavlikm/pm2-cluster-memory-cache/blob/3dde18c3d85173e7fa390387623e0b61f0910a1d/src/repositories/ProcessRepository.js#L76).

**Comment by pavlikm** [comment]
fixed in 2.1.7

**Comment by lauri865** [comment]
Amazing, thanks a lot!

Not sure if the new keyword just reduces the chance of collision or eradicates it – what broke for me was something as mundane as this (so, imagine my surprise for that error):
```typescript
nodemailer.sendMail({
    from: "sender@email.com",
    to: "random@email.com",
})
```

Actually, we unknowingly encountered that error a few months ago as well that prevented us from using another node package. We wrongly concluded that the other packages was at fault at the time. It was also not as easy to reproduce at the time, so finding the root cause was almost impossible.

So, in total, this few lines of code (magic) has caused us a productivity loss of a few days.

Best would be no polyfill altogether, if there's even a remote chance that this could happen in the future to someone with the `toItem` property.
